{"uid":"84ae5273f64db923","name":"test_result_selected_checkbox","fullName":"tests.test_live.TestLiveCodePage#test_result_selected_checkbox","historyId":"12d8e5a6f6ef0cc9c2b931edbce093ff","time":{"start":1695162133778,"stop":1695162133778,"duration":0},"status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/116.0.5845/chromedriver_linux64.zip","statusTrace":"@pytest.fixture(scope='function')\n    def driver():\n>       service = Service(ChromeDriverManager().install())\n\nconftest.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/chrome.py:39: in install\n    driver_path = self._get_driver_path(self.driver)\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/manager.py:30: in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/download_manager.py:28: in download_file\n    response = self._http_client.get(url)\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py:33: in get\n    self.validate_response(resp)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresp = <Response [404]>\n\n    @staticmethod\n    def validate_response(resp: requests.Response):\n        status_code = resp.status_code\n        if status_code == 404:\n>           raise ValueError(f\"There is no such driver by url {resp.url}\")\nE           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/116.0.5845/chromedriver_linux64.zip\n\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py:16: ValueError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1695162133779,"stop":1695162134643,"duration":864},"status":"broken","statusMessage":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/116.0.5845/chromedriver_linux64.zip\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/small_test_task_from_ritm/small_test_task_from_ritm/conftest.py\", line 11, in driver\n    service = Service(ChromeDriverManager().install())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/chrome.py\", line 39, in install\n    driver_path = self._get_driver_path(self.driver)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/manager.py\", line 30, in _get_driver_path\n    file = self._download_manager.download_file(driver.get_url())\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/download_manager.py\", line 28, in download_file\n    response = self._http_client.get(url)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py\", line 33, in get\n    self.validate_response(resp)\n  File \"/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/webdriver_manager/core/http.py\", line 16, in validate_response\n    raise ValueError(f\"There is no such driver by url {resp.url}\")\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"feature","value":"Test Live Code"},{"name":"suite","value":"Live Code"},{"name":"parentSuite","value":"tests"},{"name":"subSuite","value":"TestLiveCode"},{"name":"host","value":"fv-az344-495"},{"name":"thread","value":"2013-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_live"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"84ae5273f64db923.json","parameterValues":[]}